src/infrastructure/channels/reliable_channel.rs

```diff
@@ -141,8 +145,8 @@ impl Channel for ReliableChannel {
 
         self.dropped_packets = dropped_packets.into_iter().map(|(_, p)| p).collect();
 
-        let a = buffer.len();
-        let b = acked_header.size();
+        let _ = buffer.len();
```

Comment:
if we don't need these can we remove them? If we have them for a reason can we add a comment.